Order.Management/Shape.cs

```diff
@@ -7,11 +7,17 @@ namespace Order.Management
     abstract class Shape
     {
         public string Name { get; set; }
+
```

Comment:
standard coding practice - add new line for readability of function/method/instance declarations